lib/hanami/cli/commands/app/generate/operation.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require "dry/inflector"
+require "dry/files"
+require "shellwords"
+require_relative "../../../naming"
+require_relative "../../../errors"
+
+module Hanami
+  module CLI
+    module Commands
+      module App
+        module Generate
+          # @since x.x.x
+          # @api private
+          class Operation < App::Command
+            argument :name, required: true, desc: "Operation name"
+            option :slice, required: false, desc: "Slice name"
+
+            example [
+              %(add_book               (MyApp::Operations::AddBook)),
+              %(add_book --slice=admin (Admin::Operations::AddBook)),
```

Comment:
I wonder if we want to provide a different example here, e.g. `"books.create"` -> `(MyApp::Books::Create)`, either in addition or instead of this one.

The "operations layer" is where we really get into the "domain" of apps, and this is where I want to make sure we encourage the user to really take control of their app structure. So if they want to start organising by "concept" at this point instead of by file type, then I think we should encourage it.

What do you reckon?

Reply:
Yea, I agree.

A very small thing but how do you feel about calling it "books.add" though, instead of "books.create". 

For a publishing project, `create` would make sense (making a new book project that didn't exist before), but for a bookshelf project, the physical book already exists, it's merely being `add`ed to the bookshelf system. I feel like it's a good domain-driven design convention we can incorporate. 

Plus it can help people avoid thinking that "create" has something to do with an action or repo, since both of those often have "create". We want to let people think in terms of their domain, not the implementation.

Reply:
Yep, that's fine with me 👍🏼